### PR TITLE
Support Meta.preserve_whitespace fields list

### PR DESCRIPTION
**Pull request by alexmadjar**
DRF automatically trims whitespace on CharFields.  This can be turned off by setting the flag to False on that field.   This PR adds a Meta parameter for overriding this flag.

### CODE REVIEW

File: dynamic_rest/serializers.py [hunk 1]
```diff
@@ -549,7 +557,7 @@ def save(self, *args, **kwargs):
         return instance
 
     def id_only(self):
-        """Check whether the serializer should return an ID instead of an object.
+        """Whether the serializer should return an ID instead of an object.
```
Comment (alexmadjar) [inline dynamic_rest/serializers.py]: drive by:  makes the line length < 80

**Comment by aleontiev** [comment]
:+1:, would be good to add some documentation that surfaces this feature, e.g. in a docstring at the serializer layer, which would then be exposed in the generated HTML docs.

**Comment by alexmadjar** [comment]
@aleontiev - I don't see where we document the `Meta` api

**Comment by aleontiev** [comment]
@alexmadjar I recommend adding a section to [the DREST serializer base mixin.](https://github.com/AltSchool/dynamic-rest/blob/master/dynamic_rest/serializers.py#L120)

File: dynamic_rest/serializers.py [hunk 2]
```diff
@@ -393,6 +393,14 @@ def get_fields(self):
         ro_fields = getattr(self.Meta, 'read_only_fields', [])
         self.flag_fields(serializer_fields, ro_fields, 'read_only', True)
 
+        pw_fields = getattr(self.Meta, 'preserve_whitespace', [])
```
Comment (ryochiji) [inline dynamic_rest/serializers.py]: I'd prefer to keep the convention of ending these meta fields with `_fields` (just `preserved_whitespace` could look like it should be a boolean flag), but not sure what a non-awkward name would be. Maybe `whitespace_preserved_fields`?

**Comment by alexmadjar** [comment]
nits addressed

**Comment by ryochiji** [comment]
👍

**Pull request merged**